contracts/src/SequencerInbox.sol

```diff
@@ -59,10 +59,12 @@ contract SequencerInbox is ISequencerInbox, Initializable, UUPSUpgradeable, Owna
     }
 
     /// @inheritdoc ISequencerInbox
-    function appendTxBatch(uint256[] calldata contexts, uint256[] calldata txLengths, bytes calldata txBatch)
-        external
-        override
-    {
+    function appendTxBatch(
+        uint256[] calldata contexts,
```

Comment:
how does the gas usage change if each context is a struct (instead of embedding the tuples in the vector directly)

Reply:
I updated the gas data above with measurements for using a context struct.
Definitely a noticeable increase but idk if where to draw the line between optimizing gas cost / readability.
lmk what you think

Reply:
okay gotcha lets just continue with the current interface (no struct)